### DISCUSSION TITLE
How do I select a RadioButton on launch in WiX v4?

### DISCUSSION THREAD

**Discussion by Frittions** [wixtoolset/issues#7839] [Questions] [answered]
Hello,

**Background**:
I have a Burn bootstrapper application that has 3 radio buttons. The user needs to select one of these radio buttons and based on the selection different packages will be installed using 'InstallCondition' statements within the Bundle. 

Theme.xml
`<RadioButtons Name="RadioButtonOptions">`
`<RadioButton Name="RadioButton1" Value="0" X="205" Y="200" Width="-11" Height="17" TabStop="yes" FontId="3" HideWhenDisabled="no" HexStyle="0x000009">Option 1</RadioButton >`
`<RadioButton Name="RadioButton2" Value="1" X="205" Y="220" Width="-11" Height="17" TabStop="yes" FontId="3" HideWhenDisabled="no" HexStyle="0x000009">Option 2</RadioButton >`
`<RadioButton Name="RadioButton3" Value="2" X="205" Y="240" Width="-11" Height="17" TabStop="yes" FontId="3" HideWhenDisabled="no" HexStyle="0x000009">Option 3</RadioButton >`
`</RadioButtons>`

**Issue**:
I want the first radio button to be auto-selected when the bootstrapper application is launched, but I cannot find a setting to accomplish this.

Current:
![image](https://github.com/wixtoolset/issues/assets/97322291/53ff6f69-5bc2-4d20-93b1-849aa756004f)

Desired:
![image](https://github.com/wixtoolset/issues/assets/97322291/0bc41437-e318-4f84-ab0b-80ca8503bd9a)

**Comment by chrpai** [comment, marked as answer]
You defined a radiobutton control that uses the variable RadioButtonOptions and said that the buttons have the values 0, 1 and 2.  But you never defined the variable or gave it a value so it doesn't know which button should be selected.  You need to add this to your bundle:

    <Variable Name="RadioButtonOptions" bal:Overridable="yes" Type="string" Value="0"/>

**Reply by Frittions** [reply]
That worked! Thank you so much!
